backend/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+spring:
+  application:
+    name: cdcp-api
+  config:
+    import:
+      - optional:application-local.yml
+      - optional:application-local.yaml
```

Comment:
I would discourage doing this.. it can cause issues when running unit tests locally.

Instead, a better approach would be to require the `local` spring profile to be loaded at runtime. This can be done in the `launch.json` file, or in an eclipse `.launch` file.

Reply:
Updated VSCode's `launch,json` file to include `local` spring profile [here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/57c82567f8066b3daa2195f0ca7ad35ecbf1b414..01cc6685aed4e234740aca920fe3d426e3543e82).